src/test/order_item_summary/OrderItemSummaryController.integration.test.ts

```diff
@@ -616,11 +660,14 @@ describe("OrderItemSummaryController", () => {
             expect($($("#item-details-list .govuk-summary-list__value")[9]).text()).to.contain("Email only available for express dispatch");
             expect($($("#item-details-list .govuk-summary-list__key")[10]).text()).to.contain("Fee");
             expect($($("#item-details-list .govuk-summary-list__value")[10]).text()).to.contain("£15");
+            expect(getBasketStub).to.have.been.called;
+            expect(response.text).to.contain( "Basket (1)" )
         });
 
         it("Renders page not found if user not resource owner", async () => {
```

Comment:
Maybe we should update the title of this test, as it seems to indicate we are no longer getting `Page not found...` but `Sorry, there is a problem with the service`.